assets/js/components/notifications/CTA.js

```diff
@@ -51,7 +53,14 @@ const CTA = ( {
 				{ description }
 			</div>
 		) }
-		{ ctaLabel && (
+
+		{ inProgress && (
+			<div className="googlesitekit-cta__progress">
+				<ProgressBar />
```

Comment:
Let's add the `small` attribute to the progress bar to make it look nicer: 
```suggestion
				<ProgressBar small />
```

Reply:
👍